lib/plugins/features/movies/index.js

```diff
@@ -32,6 +33,19 @@ exports.register = (server, options, next) => {
     }
   }]);
 
+  server.route([{
+    method: 'POST',
+    path: '/movies/{id}/{location_id?}',
```

Comment:
Generally we try and have our routes be RESTful. You might be familiar with the concept [from Rails](http://guides.rubyonrails.org/v2.3/routing.html#restful-routing-the-rails-default), but its a convention we use when adding new endpoints that act on "resources" that the API exposes. [Another example of RESTful routes](https://gist.github.com/alexpchin/09939db6f81d654af06b).

Looking at a request to this endpoint (`POST /movies/1/2`), its not really clear what the second parameter is. Could you take another stab at this endpoint? A more RESTful endpoint might use [nested resources](http://guides.rubyonrails.org/v2.3/routing.html#nested-resources) or might include the location ID as a part of a standard resource update request.